src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -108,27 +99,32 @@ public static EmbedBuilder getReportEmbed(User sender, Report report) {
         StringBuilder builder = new StringBuilder("The last 5 messages by the reported user: ```\n");
         for (ReportMessage m : report.getMessages()) {
             builder.append("[").append(m.getTime().toLocalDateTime().format(DateTimeFormatter.ofPattern("HH:mm:ss"))).append(" GMT/BST] ")
-                    .append(GeneralUtils.truncate(100, m.getMessage()))
+                    .append(FormatUtils.truncate(100, m.getMessage()))
                     .append("\n");
         }
         builder.append("```");
         eb.addField("Messages from reported user", builder.toString(), false);
         return eb;
     }
 
+    /**
+     * Gets the string representing the current page
+     *
+     * @param page The current page
+     * @param items The items that are being paged
+     * @param pageLength The page length
+     * @return A string representing the current page out of the total pages. Example: 6/10
+     */
     public static String getPageOutOfTotal(int page, List<?> items, int pageLength) {
         return String.valueOf(page) + "/" + String.valueOf(items.size() < pageLength ? 1 : (items.size() / pageLength) + (items.size() % pageLength != 0 ? 1 : 0));
     }
 
-    public static String formatDuration(long duration) {
-        long totalSeconds = duration / 1000;
-        long seconds = totalSeconds % 60;
-        long minutes = (totalSeconds / 60) % 60;
-        long hours = (totalSeconds / 3600);
-        return (hours > 0 ? (hours < 10 ? "0" + hours : hours) + ":" : "")
-                + (minutes < 10 ? "0" + minutes : minutes) + ":" + (seconds < 10 ? "0" + seconds : seconds);
-    }
-
+    /**
+     * Gets the progress bar for the current {@link Track}
+     *
+     * @param track The {@link Track} to get a progress bar for
```

Comment:
Which it grabs the time from to make the progress bar (Don't use my wording, actually do good wording :p )